Externals/liblzma/liblzma.vcxproj

```diff
@@ -17,7 +17,7 @@
   <PropertyGroup Label="UserMacros" />
   <ItemDefinitionGroup>
     <ClCompile>
-      <AdditionalIncludeDirectories>.;check;common;delta;lz;lzma;rangecoder;simple;tuklib;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>.;api;check;common;delta;lz;lzma;rangecoder;simple;tuklib;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>LZMA_API_STATIC;HAVE_CONFIG_H;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
   </ItemDefinitionGroup>
```

Comment:
What's the deal with these changes? Were these incorrect beforehand?

Reply:
previously this path was added by Base.props, which was the wrong place to put it
`<AdditionalIncludeDirectories>$(ExternalsDir)liblzma\api;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>`

Reply:
Was the original goal to have some headers be public and some be private? I know visual studio seems to include zstd's `common` folder (containing e.g. `common/bitstream.h`) in the list of headers for intellisense (and it prioritizes that capitalization over dolphin's Common, though both show all files). If that was the goal originally, it didn't work, but it still seems like something that would be good to do.

Reply:
Base.props was only supposed to have the "public" headers, so this specific case (liblzma\api) was just incorrect.

The priority for build is based on order the arguments are passed on the commandline, which is a direct result of how AdditionalIncludeDirectories is formed in the xml.